scripts/windows/build/Publish-Branch.ps1

```diff
@@ -23,6 +23,8 @@ param (
     [ValidateNotNull()]
     [String] $BuildSourceVersion = $Env:BUILD_SOURCEVERSION,
 
+    [Switch] $PublishWinArm32,
```

Comment:
reminder: Same comment as the publish script for linux. Since we have removed from the build / test pipeline, we should remove this flag and the winarm32 build altogether.